gapil/analysis/analyze.go

```diff
@@ -210,6 +210,8 @@ func (s *scope) traverse(n semantic.Node) {
 	case *semantic.MapRemove:
 		// TODO: Put an entry that says the map did not contain the key.
 
+	case *semantic.MapClear:
```

Comment:
We should clear the map's entries here - there should be plenty of example test code to add a new check for this.
At the very least, please add a TODO.